RELEASE-NOTES.rst

```diff
@@ -1,8 +1,8 @@
-=======================
- Invenio-I18N v1.0.0b3
-=======================
```

Comment:
We keep the additional `=` at the end to balance the first ones. [Example](https://raw.githubusercontent.com/inveniosoftware/invenio-assets/master/RELEASE-NOTES.rst)